static/css/restyle/restyle.less

```diff
@@ -1774,6 +1773,11 @@ h3.author .transfer-ownership {
   .addon.hovercard & {
     border: 1px solid #ebebeb;
   }
+
+  .more-versions {
+    color: #0996f8;
+    margin: 0 .25em 0 0.20em;
```

Comment:
Even I was a bit in dilemma to use that, actually I just referred from `.button span` CSS rule `0 .25em 0 -.25em`
I will put the same in px now